### ISSUE TITLE
When predict(), it returns disconnected graph error

### ISSUE THREAD

**Issue by mhyeonsoo** [4uiiurz1/keras-arcface#18]
Hello, 
I am struggling in solving this issue to get the prediction result.
The code I am using now is below.
~~~
arcface_model = Model(inputs=arcface_model.input[0], outputs=arcface_model.output)     
~~~

This part returns me the error like

**Graph disconnected: cannot obtain value for tensor Tensor("input_1:0", shape=(None, 200), dtype=float32) at layer "input_1". The following previous layers were accessed without issue: ['input_2', 'resnet50', 'batch_normalization', 'dropout', 'flatten', 'dense', 'batch_normalization_1']**

And the model summary is below as well.
![image](https://user-images.githubusercontent.com/32179857/82849728-266cc600-9f34-11ea-9c1c-3958c2f769dc.png)

can you help me to figure this out?

Thanks a lot.
